src/compose/network-manager.ts

```diff
@@ -10,16 +9,11 @@ import log from '../lib/supervisor-console';
 import * as logger from '../logger';
 import { Network } from './network';
 import { ResourceRecreationAttemptError } from './errors';
+import { NetworkInspectInfo } from './types/network';
 
-export function getAll(): Bluebird<Network[]> {
-	return getWithBothLabels().map((network: { Name: string }) => {
-		return docker
-			.getNetwork(network.Name)
-			.inspect()
-			.then((net) => {
-				return Network.fromDockerNetwork(net);
-			});
-	});
+export async function getAll(): Promise<Network[]> {
+	const dockerNets = await getWithBothLabels();
+	return dockerNets.map((net) => Network.fromDockerNetwork(net));
```

Comment:
The `inspect()` is superfluous because `getNetwork` already returns the needed network info.